lib/index.ts

```diff
@@ -350,6 +350,7 @@ const serialize = (
     clientId,
     transport,
     nsp,
+    data: socket.data,
```

Comment:
We add an `_admin` attribute [here](https://github.com/socketio/socket.io-admin-ui/blob/9c8e7728a23db8f328848a9a05ae4ea17ab0f4ad/lib/index.ts#L306-L309), would it make sense to remove this attribute before sending the data to the user?